terraform/environments/data-platform/api.tf

```diff
@@ -131,9 +128,6 @@ resource "aws_api_gateway_integration" "ingest_data_for_data_product_to_lambda"
   uri                     = module.data_product_presigned_url_lambda.lambda_function_invoke_arn
 
   request_parameters = {
-    "integration.request.querystring.database"   = "method.request.querystring.database",
-    "integration.request.querystring.table"      = "method.request.querystring.table",
-    "integration.request.querystring.contentMD5" = "method.request.querystring.contentMD5",
```

Comment:
Just wondering, do we need to redeclare these as body parameters? 

I'm not all that familiar with API gateway so don't know how is uses these definitions, I just saw you can specify body params as well via JSONPath https://docs.aws.amazon.com/apigateway/latest/developerguide/request-response-data-mappings.html